test/openAPI/features/g2p_prepayment_validation.feature

```diff
@@ -0,0 +1,169 @@
+        Then The response from the /prepayment-validation is received
+        And The /prepayment-validation response should be returned in a timely manner 15000ms
+        And The /prepayment-validation response should have status 400
+        And The /prepayment-validation response should have content-type: application/json header
+        And The /prepayment-validation response should match json schema
+        And The /prepayment-validation response ResponseCode field should be "01"
+
+    @unit @negative
+    Scenario: Unable to retrieves eligible Functional IDs from the account mapper for credit transfers because of missing required Currency in the payload
+        Given Wants to retrieves eligible Functional IDs from the account mapper for credit transfers
+        When POST request with required body with given "abcdef123456" as RequestID, "sourceBBID12" as SourceBBID, "batchID12345" as BatchID, "instructionID123" as InstructionID, "PayeeFunctionalID123" as PayeeFunctionalID, "100" as Amount, "Narration" as Narration is sent
+        Then The response from the /prepayment-validation is received
+        And The /prepayment-validation response should be returned in a timely manner 15000ms
+        And The /prepayment-validation response should have status 400
+        And The /prepayment-validation response should have content-type: application/json header
+        And The /prepayment-validation response should match json schema
+        And The /prepayment-validation response ResponseCode field should be "01"
+
+    @unit @negative
+    Scenario: Unable to retrieves eligible Functional IDs from the account mapper for credit transfers because of missing required Narration in the payload
+        Given Wants to retrieves eligible Functional IDs from the account mapper for credit transfers
+        When POST request with required body with given "abcdef123456" as RequestID, "sourceBBID12" as SourceBBID, "batchID12345" as BatchID, "instructionID123" as InstructionID, "PayeeFunctionalID123" as PayeeFunctionalID, "100" as Amount, "USD" as Currency is sent
+        Then The response from the /prepayment-validation is received
+        And The /prepayment-validation response should be returned in a timely manner 15000ms
+        And The /prepayment-validation response should have status 400
+        And The /prepayment-validation response should have content-type: application/json header
+        And The /prepayment-validation response should match json schema
+        And The /prepayment-validation response ResponseCode field should be "01"
+
+    @unit @negative
+    Scenario: Unable to retrieves eligible Functional IDs from the account mapper for credit transfers because of invalid SourceBBID paramater in payload
+        Given Wants to retrieves eligible Functional IDs from the account mapper for credit transfers
+        When POST request with required body with given "sourceBBID" as invalid SourceBBID parameter is sent
```

Comment:
I would add maybe requestID also to this when so we can check in the end if response's RequesID matches with the request's RequestID

Reply:
My fault, because in negative scenarios when receive status code 400 there is no response like in the bulk payment. We don't have it currently in the API spec but it should be the same like in the bulk payments